rep-2005.rst

```diff
@@ -50,6 +50,22 @@ To propose changes, such as adding or removing items, make a pull request to tha
 The ensuing public review and discussion will evaluate the change in light of the purpose and scope explained above, with the final merge decision being made by one of the REP maintainers.
 
 
+Schedule of changes
+-------------------
+
+To facilitate understanding of what items make up the integral list of packages for a distribution, accepted changes may only be merged on a fixed schedule, detailed below.
+
+* When the code freeze for distribution A occurs:
+  * the version of this list at that time becomes the *definitive* list for that distribution,
+  * the list becomes open for accepting changes, and
+  * a review of the current content is conducted to ensure all packages still meet the criteria for being on the list.
+* When the development of distribution B begins:
+  * the version of this list at that time becomes the *proposed* list for that distribution, and
+  * additions may no longer be made, but removals are still possible until the code freeze date.
```

Comment:
Again, maybe I misunderstand the idea. E.g. for G-turtle the development starts roughly after Foxy is released. Are you suggesting that in May 2020 we are already "locking" the list (in terms of only removing items) for G-turtle scheduled for May 2021? That sounds like a very long time ahead to me.